### PR TITLE
Bug 2099763: update icons for eventSource and eventSink

### PR DESCRIPTION
**Pull request by invincibleJai**
**Tracks:**
 https://bugzilla.redhat.com/show_bug.cgi?id=2099763
 https://issues.redhat.com/browse/ODC-6643

**Descriptions:**

Update the Event Sink and Event Source icons based on https://issues.redhat.com/browse/DTUX-1212 

Screenshots:

image

image

image

### CODE REVIEW

**Comment by invincibleJai** [comment]
/cc @serenamarie125
/cc @jerolimov 
/cc @vikram-raj 


PTAL when you get time

**Comment by jerolimov** [comment]
@invincibleJai @serenamarie125 @beaumorley

IMHO this icons should center-align the circle center... Esp. on the left side of the name this looks really unaligned.

image

image

File: frontend/packages/knative-plugin/src/utils/icons.tsx [hunk 1]
```diff
@@ -1,9 +1,23 @@
...
-import { SignInAltIcon, SignOutAltIcon } from '@patternfly/react-icons';
 import * as channelIcon from '../imgs/channel.svg';
+import * as eventSinkIcon from '../imgs/event-sink.svg';
+import * as eventSourceIcon from '../imgs/event-source.svg';
 
-export const eventSourceIcon = <SignOutAltIcon />;
+const eventIconStyle = {
+  width: '1em',
+  height: '1em',
+};
```
Comment (jerolimov) [inline frontend/packages/knative-plugin/src/utils/icons.tsx:10]: It looks like this width and height breaks the icons mentioned above (icon on the left side of a name)

```suggestion
```

See:

image

vs:

image

**Review by jerolimov — CHANGES_REQUESTED** [review]
see above

File: frontend/packages/knative-plugin/src/utils/icons.tsx (hunk 1 above)
Comment (jerolimov) [reply frontend/packages/knative-plugin/src/utils/icons.tsx:10]: @invincibleJai here are two icons where I moved the inner circle-center to the center of the image. PTAL. :)

event-sink

event-source

Comment (invincibleJai) [reply frontend/packages/knative-plugin/src/utils/icons.tsx]: Thanks @jerolimov, have updated it PTAL

image

image

**Review by jerolimov — APPROVED** [review]

**Pull request merged**